mlc_llm/relax_model/mistral.py

```diff
@@ -0,0 +1,1066 @@
+# pylint: disable=too-many-lines, missing-class-docstring, missing-function-docstring
+"""Implements the mistal model with sliding window attention."""
+
```

Comment:
We need to migrate into the new compiler convention soon, see https://github.com/mlc-ai/mlc-llm/tree/main/python/mlc_chat/compiler/model